docs/src/components/__tests__/TopNavigation.test.js

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import TopNavigation from '../TopNavigation';
+
+describe('Header', () => {
+	it('renders header', () => {
+		const data = [
+			{
+				name: 'home',
+				path: '/',
+				label: 'Home',
+				children: [],
+			},
+			{
+				name: 'design-tokens',
+				path: '/design-tokens/',
+				label: 'Design Tokens',
+				children: [
+					{
+						name: 'design-tokens',
+						path: '/design-tokens/',
+						label: 'Design Tokens',
+						children: [],
+					},
+				],
+			},
+			{
+				name: 'components',
+				path: '/components/',
+				label: 'Components',
+				children: [
+					{
+						name: 'components',
+						path: '/components/',
+						label: 'Components',
+						children: [],
+					},
+					{
+						name: 'autocomplete',
+						path: '/components/autocomplete',
+						label: 'Autocomplete',
+						children: [],
+					},
+				],
+			},
+		];
+		const pathSplit = ['components'];
+		render(<TopNavigation data={data} path={pathSplit} />);
```

Comment:
There should be a test here that makes sure the order is correct. (i.e. more than 1 component.)